stl/debugger/STL.natvis

```diff
@@ -1665,9 +1665,8 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       <Expand/>
   </Type>
 
-
-  <!-- VC 2015 -->
-  <Type Name="std::recursive_mutex">
+  <Type Name="std::mutex">
+      <AlternativeType Name="std::recursive_mutex&lt;*&gt;"/>

```

Comment:
Why `&lt;*&gt;`? Is recursive mutex a template?

Reply:
this was an accident removing it